lib/Client.js

```diff
@@ -7,9 +7,22 @@ var min = require('osc-min');
 var Message = require('./Message');
 
 var Client = function (host, port) {
+    var isBroadcast = host.split('.').reduce(function(previous, current){
+        return previous ? true : +current === 255;
+    }, false),
+    setter = function () {
+        this.setBroadcast(isBroadcast);
```

Comment:
ah, before I forget: yes that would be nice but at this point the socket is not bound yet. setting the broadcast flag is only possible after initialization though. so I used this as a workaround. (some more info here: https://stackoverflow.com/questions/9242957/node-js-dgram-setbroadcastflag-fails-due-to-ebadf)